components/Article/Page.js

```diff
@@ -736,10 +723,8 @@ class ArticlePage extends Component {
           meta && meta.discussionId && router.query.focus ? undefined : meta
         }
         onNavExpanded={this.onPrimaryNavExpandedChange}
-        secondaryNav={seriesNavButton || (!isTester && actionBarNav)}
-        showSecondary={
-          isTester && seriesNavButton ? true : this.state.showSecondary
-        }
+        secondaryNav={seriesNavButton}
+        showSecondary={seriesNavButton ? true : this.state.showSecondary}
```

Comment:
Is `this.state.showSecondary` still necessary? IMO can be removed (also from initial state)